new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
```

Comment:
Unneeded change?

Reply:
I'm confused why this didn't go away. Should there be an extra space between the closing bracket on 1083 and the `return link;` line?

Reply:
I see a small whitespace on the line below.
It looks like you may have added two space characters on this line?
You can also check this locally by looking at the git diff in the command line or in the Github application.

Reply:
I think I've fixed this now? I removed the space characters, but your comment wasn't hidden so I'm not sure.